website/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@equalstreetnames/website",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openknowledgebe/equalstreetnames.git"
+  },
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/openknowledgebe/equalstreetnames/issues"
+  },
+  "homepage": "https://github.com/openknowledgebe/equalstreetnames#readme",
```

Comment:
Should probably be changed to `https://github.com/EqualStreetNames/equalstreetnames#readme`